tests/optimization/bfgs_05.py

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
add a comment what this file is for (e.g. comparison for bla.cc)
add copyright?